config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -39,6 +39,17 @@ spec:
           spec:
             description: SSPSpec defines the desired state of SSP
             properties:
+              commonInstancetypes:
+                description: CommonInstancetypes is the confgiruation of the common
+                  instance types operand
+                properties:
+                  uri:
```

Comment:
What is the URI? Does it have to be http/https? Would it work with a github repo, or a plain git repo?

Reply:
Updated with more context.